### DISCUSSION TITLE
python error

### DISCUSSION THREAD

**Discussion by amphy92** [bckd00r/LViewPlus#37] [General]
hey there, i have a python error.
Fatal Python error: init_fs_encoding: failed to get the Python codec of the filesystem encoding
Python runtime state: core initialized
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00003b08 (most recent call first):
<no Python frame>

but python is already installed and in the default path. 

any fixes?

**Comment by Azizmire** [comment]
Header-title {
  font-size: 1.75rem;
}

.Cards {
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
}
.Card {
  max-width: 40rem;
  flex: 1;
}
.Card-inner {
  display: flex;
  flex: 1;
  flex-direction: column;
  padding: 0rem;
}
